Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apk add --update --no-cache \
 
 COPY --from=ethereum/solc:0.5.17 /usr/bin/solc /usr/bin/solc
 
-RUN go get gotest.tools/gotestsum
+RUN go install gotest.tools/gotestsum@latest
```

Comment:
Should we also update:

https://github.com/keep-network/keep-core/blob/b872c35aa405b4e857fba7317aa6bd23137f5a01/scripts/macos-setup.sh#L18-L23

https://github.com/keep-network/keep-core/blob/b872c35aa405b4e857fba7317aa6bd23137f5a01/scripts/ubuntu-setup.sh#L107

?

Reply:
@pdyraga 
We should, see https://github.com/keep-network/keep-core/pull/3035.